tests/layer1/test_layer1_speed_Novus_10G.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+from abstract_open_traffic_generator.port import *
+from abstract_open_traffic_generator.config import *
+from abstract_open_traffic_generator.layer1 import *
+from abstract_open_traffic_generator.control import *
+
```

Comment:
most changes requested in Novus100G script applies here as well.

Reply:
updated